wdl2cwl/tests/cwl_files/CollectQualityYieldMetrics.cwl

```diff
@@ -15,17 +15,16 @@ outputs:
 requirements:
 - class: DockerRequirement
   dockerPull: us.gcr.io/broad-gotc-prod/picard-cloud:2.23.8
+- class: InitialWorkDirRequirement
+  listing:
+  - entryname: example.sh
+    entry: 'java -Xms2000m -jar /usr/picard/picard.jar \CollectQualityYieldMetrics
+      \INPUT=$(inputs.input_bam.path) \OQ=true \OUTPUT=$(inputs.metrics_filename) '
```

Comment:
Almost! There are extra `\` in here, corresponding to the end of the line in the original `command` block. Can you remove those `\`?